ops/testing.py

```diff
@@ -176,47 +176,49 @@ class Harness(Generic[CharmType]):
 
     The model created is from the viewpoint of the charm that is being tested.
 
-    Below is an example test using :meth:`begin_with_initial_hooks` that ensures
-    the charm responds correctly to config changes::
+    Always call ``harness.cleanup()`` after creating a :class:`Harness`::
```

Comment:
I like how you've introduced this here - it's sold me on this approach, I think.